src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -2369,6 +2375,23 @@ export class StudyViewPageStore {
 
         const cancerTypeIds = _.uniq(this.queriedPhysicalStudies.result.map(study => study.cancerTypeId));
 
+        this.chartsType.set(UniqueKey.OVERALL_SURVIVAL, ChartTypeEnum.SURVIVAL);
+        this.chartsDimension.set(UniqueKey.OVERALL_SURVIVAL, STUDY_VIEW_CONFIG.layout.dimensions[ChartTypeEnum.SURVIVAL]);
+        if (osStatusFlag && osMonthsFlag && getDefaultPriorityByUniqueKey(UniqueKey.OVERALL_SURVIVAL) !== 0) {
+            // hide OVERALL_SURVIVAL chart if cacner type is mixed or have moer than one cancer type
```

Comment:
Look good to me!! Just a small thing, there is a typo in the comments for `more` at this line and Line 2389.